netx/net/sourceforge/jnlp/util/WindowsDesktopEntry.java

```diff
@@ -59,7 +59,7 @@ public File getDesktopIconFile() {
     public void createShortcutOnWindowsDesktop() throws IOException {
         String path = getDesktopLnkPath();
         String JavaWsBin = XDesktopEntry.getJavaWsBin();
-        ShellLink sl = ShellLink.createLink(JavaWsBin).setCMDArgs(file.getSourceLocation().toString());
+        ShellLink sl = ShellLink.createLink(JavaWsBin).setCMDArgs('"' + file.getSourceLocation().toString() + '"');
```

Comment:
const char?

Reply:
or const String is better ? becaus we concat with Strings

Reply:
private static final String DOUBLE_QUOTE = "\"";
?

Reply:
or a helper method
```
private String quoted(URL url) {
    return String.format("\"%s\"", url.toString());
}
```

And maybe call `toExternalForm()` instead of `toString()`

Reply:
jdk8 impl from toString()
>    public String toString() {
>        return toExternalForm();
>    }

Reply:
Yes, I simply find `toExternalForm()` more explicit. And I don't like to rely too much onto the implementation of toString() as for my understanding the main purpose of toString is to have a human readable representation of the object and not to meet some business requirements...